### DISCUSSION TITLE
Extract Cuisine when available, and add separate section for it

### DISCUSSION THREAD

**Discussion by epheterson** [mealie-recipes/mealie#1400] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [ X ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [ X ] Reviewed the existing feature requests and my feature has not been requested.
- [ X ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Many online recipes explicitly label a cuisine, though in Mealie there's currently no way to store this info outside of custom Categories.

**(Optional): Suggest A Solution**
It'd be great if Mealie added proper Cuisine support. A current workaround is to use custom tags, though a separate field for Cuisine would be helpful. When a recipe has cuisine data clearly labeled, it'd be great to also extract this and include it in the recipe.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by hay-kot** [comment]
I don't really see a purpose in this since we already have robust Tag and Category support. Adding Cuisine would just be a duplication of the behavior in both Tags and Categories which would be a significant amount of work for what could be accomplished by a Category.

**Comment by epheterson** [comment]
Fair view, I think there's still an argument that it'd be useful to automatically extract cuisine into a category/tag label (when available).
